tests/publiccloud/run_ltp.pm

```diff
@@ -48,38 +48,13 @@ sub instance_log_args
 sub upload_ltp_logs
 {
     my ($self) = @_;
-    my $ltp_testsuite = get_required_var('LTP_COMMAND_FILE');
-    my $log_file = Mojo::File::path('ulogs/result.json');
     record_info('LTP Logs', 'upload');
-    upload_logs("$root_dir/result.json", log_name => $log_file->basename, failok => 1);
+    assert_script_run("test -f $root_dir/result.json || echo No result log");
+    parse_extra_log('LTP', "$root_dir/result.json");
     # debug file in the standart LTP log-dir. structure:
     assert_script_run("test -f /tmp/runltp.\$USER/latest/debug.log || echo No debug log");
     upload_logs("/tmp/runltp.\$USER/latest/debug.log", failok => 1);
 
-    return unless -e $log_file->to_string;
-
-    local @INC = ($ENV{OPENQA_LIBPATH} // testapi::OPENQA_LIBPATH, @INC);
-    eval {
-        require OpenQA::Parser::Format::LTP;
-
-        my $ltp_log = Mojo::JSON::decode_json($log_file->slurp());
-        my $parser = OpenQA::Parser::Format::LTP->new()->load($log_file->to_string);
-        my %ltp_log_results = map { $_->{test_fqn} => $_->{test} } @{$ltp_log->{results}};
-        my $whitelist = LTP::WhiteList->new();
-
-        for my $result (@{$parser->results()}) {
-            if ($whitelist->override_known_failures($self, {%{$self->{ltp_env}}, retval => $ltp_log_results{$result->{test_fqn}}->{retval}}, $ltp_testsuite, $result->{test_fqn})) {
-                $result->{result} = 'softfail';
-            }
-        }
```

Comment:
Using `parse_extra_log` only, you missed all this **softfail** section, not present inside that routine.

Moreover, if you remove all this `eval{}`, you should remove the `die $@ if $@` too, here below left.

Reply:
sorry that I was too fast . @mimi1vx  maybe we need to revert this part which @m-dati mentioning ?